nose_launchable/client.py

```diff
@@ -15,8 +18,19 @@ class LaunchableClientFactory:
     @classmethod
     def prepare(cls):
         url, org, wp, token = cls._parse_options()
+        strategy = Retry(
+            total=3,
+            method_whitelist=["GET", "PUT", "PATCH", "DELETE"],
+            status_forcelist=[429, 500, 502, 503, 504],
```

Comment:
I think we shouldn't retry when we caught 500 and 503 errors. What do you think?

Reply:
Hmm, I personally don't see any problems when retrying a request when the server returns 500 or 503. Those methods are supposed to be idempotent so why don't we retry the request when we encounter any problems if there is any chance to succeed?

Reply:
The reason came from only my experience... 
So after I commented, I researched that which status code should retry and which status code should not retry. Finally, I'll agree with these status codes. 

Because if 4xx error happened, the cause is almost the client. On the other hand, if caught 5xx error, the error came from server sides. so the client should retry the request.